tiller_python_service/controller.py

```diff
@@ -17,6 +17,7 @@
 
 
 def is_valid_python(code):
+    print("Checking if valid python")
```

Comment:
We should be using the `logging` library here, can we use that instead of prints?

Reply:
We should be using the logging library here, can we use that instead of prints? help tiller